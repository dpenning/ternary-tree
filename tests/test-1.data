(36 535 369 837 738 110 817 240 478 483 667 436 186 323 782 256 64 977
815 126 701 888 46 350 830 689 187 833 531 776 979 147 856 949 901 189
92 981 572 307 821 397 911 776 737 899 872 224 674 752 820 541 680 496
532 461 483 642 172 316 312 1 940 957 589 217 671 114 244 109 232 912
575 259 67 86 413 722 540 140 441 791 412 823 492 54 956 572 570 801 15
974 674 529 48 563 748 523 717 10 622 278 229 625 533 880 14 298 992 200
928 210 744 101 172 592 496 472 294 224 678 249 709 438 895 152 129 999
973 354 532 976 669 965 837 585 665 140 574 111 36 701 451 319 114 869
985 49 767 682 215 839 932 645 814 730 187 597 767 856 960 654 642 828
948 667 714 275 623 187 804 535 284 452 261 473 734 982 699 567 431 867
613 858 891 355 153 450 508 881 65 147 161 850 986 401 339 167 53 145)