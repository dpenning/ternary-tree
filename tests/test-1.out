
((NIL 1 (10 14) 15 (36)) 36
 ((NIL 46 (NIL 48 (49 53) 54 NIL) 64 (((65) 67 NIL 86 NIL) 92 (101) 109 NIL))
  110
  (((NIL 111 (114) 114 NIL) 126 (NIL 129 (140) 140 (145 147)) 147
    ((NIL 152 NIL 153 (161 167)) 172 NIL 172 NIL))
   186
   ((187 187) 187 NIL 189 ((NIL 200 NIL 210 (215)) 217 (224) 224 (229 232)))
   240
   ((244 249) 256
    ((NIL 259 (261 275) 278 ((284) 294 NIL 298 NIL)) 307 (312) 316 (319)) 323
    ((339) 350 NIL 354 (355))))
  369
  ((NIL 397 ((401) 412 NIL 413 (431)) 436
    ((438) 441 ((450) 451 NIL 452 NIL) 461 (472 473)))
   478 (483) 483
   ((492 496) 496 ((508) 523 NIL 529 NIL) 531 ((532) 532 NIL 533 (535)))))
 535
 ((((540) 541 ((563 567) 570 NIL 572 NIL) 572
    (((574) 575 NIL 585 NIL) 589 ((NIL 592 NIL 597 (613)) 622 (623) 625 (642))
     642 (NIL 645 (654) 665 (667))))
   667 (((669) 671 NIL 674 NIL) 674 (NIL 678 NIL 680 (682)) 689 (699 701)) 701
   ((NIL 709 (714) 717 NIL) 722 (730 734) 737 NIL))
  738
  ((((744) 748 NIL 752 (767 767)) 776 NIL 776 NIL) 782
   (NIL 791 (NIL 801 (804) 814 NIL) 815 NIL) 817
   ((NIL 820 NIL 821 (823 828)) 830 NIL 833 (837)))
  837
  (((NIL 839 (850) 856 NIL) 856 ((858) 867 NIL 869 NIL) 872 (880 881)) 888
   (((891) 895 NIL 899 NIL) 901 (NIL 911 (NIL 912 NIL 928 (932)) 940 (948)) 949
    (NIL 956 NIL 957 ((960 965) 973 NIL 974 (976))))
   977 (NIL 979 NIL 981 ((NIL 982 NIL 985 (986)) 992 NIL 999 NIL))))) 
